locales-pending/onboarding.ftl

```diff
@@ -0,0 +1,19 @@
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+onboarding-steps-get-started-label = Get started
+onboarding-steps-enter-info-label = Enter info
+onboarding-steps-find-exposures-label = Find exposures
+onboarding-get-started-heading = Welcome to { -product-short-name }. Let’s find your exposed information.
+onboarding-get-started-content-data = To find and fix your exposed data, we’ll need to ask for your personal info. Rest assured we will safeguard your information like it’s our own.
+onboarding-get-started-content-price = We believe in your right to privacy, so basic { -product-short-name } is always free. Upgrade to { -brand-premium } for continuous protection and automatic removal of your personal info.
+onboarding-get-started-content-explainer = See how we protect your data
+onboarding-get-started-cta-label = Start my free scan
+onboarding-get-started-how-it-works-dialog-heading-line1 = We’re here to protect your personal information
+onboarding-get-started-how-it-works-dialog-heading-line2 = Here’s how it works:
+onboarding-get-started-how-it-works-dialog-step1-heading = 1. Scan for exposures
+onboarding-get-started-how-it-works-dialog-step1-content = With just a few key pieces of information, we’ll search for you in all known data breaches and 190 major data broker sites. All users get the first scan free.
+onboarding-get-started-how-it-works-dialog-step2-heading = 2. Remove your personal info
+onboarding-get-started-how-it-works-dialog-step2-content = We’ll show you what personal info each data broker site is selling. You’ll need to contact each data broker to request removal. Removal typically takes 7-10 business days, but some can take longer. Upgrade to { -brand-premium } and we’ll handle this for you.
+onboarding-get-started-how-it-works-dialog-step3-heading = 3. Resolve data breaches
+onboarding-get-started-how-it-works-dialog-step3-content = We’ll give you clear steps on how to resolve each data breach. They’ll require access to your accounts, so you’ll need to resolve them manually even if you upgrade to { -brand-premium }.
```

Comment:
Yeah, I was kinda going back and forth on this one, because it didn't really feel like an enumeration/I wasn't sure how important the order was. Though looking back at it now maybe it does. Happy to change to an `<ol>`.

Reply:
Turned into an `<ol>` in 53b248b52, thanks for the callout. Then we can also rely on the browser to localise the numbering, if needed.